pkg/deploy/graph/analysis/dc.go

```diff
@@ -35,6 +38,24 @@ dc:
 				// The image stream for the tag of interest does not exist.
 				// TODO: Suggest `oc create imagestream` once we have that.
 				if isNode, exists := doesImageStreamExist(g, uncastIstNode); !exists {
+
+					// We may not be able to see the image stream because it's in different project.
+					for _, t := range dcNode.DeploymentConfig.Spec.Triggers {
+						if t.Type == deployapi.DeploymentTriggerOnImageChange &&
+							t.ImageChangeParams.From.Namespace != dcNode.DeploymentConfig.Namespace {
+							markers = append(markers, osgraph.Marker{
+								Node:         uncastDcNode,
+								RelatedNodes: []graph.Node{uncastIstNode, isNode},
+
+								Severity:   osgraph.WarningSeverity,
```

Comment:
We don't do anything with info so I am going to totally remove this. I am thinking of a simple warning at the end to run with --chase-refs if there are any cross-namespace links.